regulations/views/comment.py

```diff
@@ -11,6 +12,13 @@
 from regulations import tasks
 import requests
 
+# The following limit is specified by the regulations.gov API
+# It is not available as a queryable endpoint
+VALID_ATTACHMENT_EXTENSIONS = set([
```

Comment:
I'm just hairsplitting here, but now we're defining some file rules here and another (max size) in settings. I don't particularly care where they go, but they might as well live in the same place.